src/lib/expansion/expansion-panel-header.scss

```diff
@@ -0,0 +1,60 @@
+$md-expansion-panel-header-height: 48px;
+$md-expansion-panel-header-height-expanded: 64px;
+
+.mat-expansion-panel-header {
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  height: $md-expansion-panel-header-height;
+  line-height: $md-expansion-panel-header-height;
+  padding: 0 24px;
+
+  .mat-content {
+    display: flex;
+    flex: 1;
+    flex-direction: row;
+    overflow: hidden;
+  }
+
+  &.mat-expanded {
+    height: $md-expansion-panel-header-height-expanded;
+    line-height: $md-expansion-panel-header-height-expanded;
+  }
+
+  &:focus, 
+  &:hover {
+    background: rgb(238, 238, 238);
+    outline: none;
+  }
+
+  &.mat-expanded:focus,
+  &.mat-expanded:hover, {
+    background: inherit;
+  }
+
+  [title] {
```

Comment:
Prefer class selectors over attribute selectors (especially for such common attributes as `title` and `description`)